### PR TITLE
feat(content): read cheatsheet content from local file

### PR DESCRIPTION
**Pull request by petebacondarwin**
We don't want the cheatsheet from the angular.io project. We want to read a local file and use that instead.

@wardbell - how about this?

### CODE REVIEW

File: processors/WriteFiles.ts [hunk 1]
```diff
@@ -21,8 +21,10 @@ export class WriteFilesProcessor implements Processor {
         if (doc.intro) {
           renderedContents += '@intro\n' + doc.intro + '\n\n';
         }
-        if (doc.renderedAST) {
+        if (doc instanceof PugDocument) {
```
Comment (duchuyctlk) [inline processors/WriteFiles.ts:24]:
```javascript
renderedContents += '@description\n' + (doc instanceof PugDocument ? doc.renderedAST : doc.contents);
```

Comment (petebacondarwin) [reply processors/WriteFiles.ts:24]: indeed.

**Comment by wardbell** [comment]
I approve of the approach.

The implementation seems right but I'm not that close to it and so am unreliable on that score.

**Comment by petebacondarwin** [comment]
Thanks @wardbell

**Pull request merged**